docs/development/contributor_guidelines.rst

```diff
@@ -118,16 +118,35 @@ should be removed. If you are unfamiliar with how to squash commits with rebase,
 check out this
 `blog post <http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html>`__.
 
+.. _contributor-permissions:
+
 Privileges
 ----------
 
+.. note:: the translation workflow is different from the code and
+          documentation development workflow, see the associated
+          :ref:`privilege escalation <i18n-administrator-permissions>`
+          documentation.
```

Comment:
This section (and the corresponding Note over in the i18n docs) could be worded more clearly—I'll add some suggestions.